stdlib/universe/difference.go

```diff
@@ -281,7 +302,11 @@ func (t *differenceTransformation) Process(id execute.DatasetID, tbl flux.Table)
 							return err
 						}
 					} else {
-						if err := builder.AppendNil(j); err != nil {
+						if i == 0 && t.initialZero {
+							if err := builder.AppendFloat(j, float64(0)); err != nil {
```

Comment:
Nitpick: `float64(0)` is not needed here. As the function takes a float, Go will assume the constant is a float.